app/services/integrations/aggregator/taxes/credit_notes/create_service.rb

```diff
@@ -4,11 +4,11 @@ module Integrations
   module Aggregator
     module Taxes
       module CreditNotes
-        class CreateService < Integrations::Aggregator::Taxes::Invoices::BaseService
+        class CreateService < Integrations::Aggregator::Taxes::BaseService
           def initialize(credit_note:)
             @credit_note = credit_note
 
-            super(invoice: credit_note.invoice)
+            super(integration:)
```

Comment:
since integration is defined at base service, what do you think if here we will call super without arguments, and will call 
`super(integration:)` form the `Integrations::Aggregator::Taxes::BaseService`, where it's defined?

Reply:
(it's just a matter of taste, so if you don't like it feel free to ignore)

Reply:
For me it is the same, but if you have this preference, here it is:)